docs/content/2.components/CldOgImage.md

```diff
@@ -27,9 +27,21 @@ The resulting HTML will be applied to the Head of the document:
 <meta content="https://res.cloudinary.com/nuxt-cloudinary/image/upload/c_fill,w_2400,h_1254,g_center/c_scale,w_1200/f_jpg/q_auto/cld-sample-2?_a=BBDAACAD0" property="og:image:secure_url">
 <meta content="1200" property="og:image:width">
 <meta content="627" property="og:image:height">
-<meta content=" " property="twitter:title" />
+<meta content="My awesome Twitter Title" property="twitter:title" />
 <meta content="summary_large_image" property="twitter:card">
 <meta content="https://res.cloudinary.com/nuxt-cloudinary/image/upload/c_fill,w_2400,h_1254,g_center/c_scale,w_1200/f_webp/q_auto/cld-sample-2?_a=BBDAACAD0" property="twitter:image">
 ```
 
+Keep in mind that if you want to render an OG image in Twitter, you need to:
+
+1. Have a `currentRoute.meta?.title` set to some value
+2. Pass a prop to the component like following:
+
+```html
+<CldOgImage
+  src="cld-sample-2"
+  twitter-title="My Awesome Twitter Title"
```

Comment:
should this be `twitterTitle` or is that a Nuxt convention to turn it to kebab case?

Reply:
guessing thats the case based on other examples im seeing

Reply:
Vue uses different naming for registering and using props.

So in this case the prop name is actually `twitterTitle` but how we use it in Vue component is like `twitter-title`.